src/assets/translations/en/main.json

```diff
@@ -958,6 +958,21 @@
       "body": "By connecting a wallet, you agree to ShapeShift's Terms of Service and acknowledge that you have read and understand the ShapeShift disclaimer.",
       "footer": "Don't have a wallet?",
       "create": "Create one"
+    },
+    "walletConnect": {
+      "errors": {
```

Comment:
This is missing the `rpcUrlNotFound` translation

Reply:
👍